src/SFA.DAS.EAS.Web/ViewModels/Transactions/TransferSenderTransactionDetailsViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using SFA.DAS.EAS.Domain.Models.Transfers;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Web.ViewModels.Transactions
+{
+    public class TransferSenderTransactionDetailsViewModel : ViewModelBase
```

Comment:
`ViewModelBase` is deprecated (we should tag it with the Obsolete attribute) and `ViewModel` should be used instead.